api/tests/cond.py

```diff
@@ -0,0 +1,68 @@
+#   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from common_import import *
+
+
+class PDCond(PaddleAPIBenchmarkBase):
+    def build_program(self, config):
+        with fluid.program_guard(self.main_program, self.startup_program):
+
+            def true_fn():
+                return fluid.layers.fill_constant(
```

Comment:
When you just add fill_constant ... whether the benchmark will have significant time consumption for you to analyze ... ?